Author: Yves-Alexis Perez <corsac@debian.org>
Author: Lionel Le Folgoc <mrpouit@gmail.com>
Description: adapt config to Debian
Especially:

* use X11 session wrapper
* start on vt7
* don't show userlist

Forwarded: not-needed

=== modified file 'data/lightdm.conf'
Index: lightdm-1.2.0/data/lightdm.conf
===================================================================
--- lightdm-1.2.0.orig/data/lightdm.conf	2012-04-04 07:17:25.000000000 +0200
+++ lightdm-1.2.0/data/lightdm.conf	2012-04-05 19:10:50.378808312 +0200
@@ -58,18 +58,18 @@
 #xserver-command=X
 #xserver-layout=
 #xserver-config=
-#xserver-allow-tcp=false
+xserver-allow-tcp=false
 #xdmcp-manager=
 #xdmcp-port=177
 #xdmcp-key=
-#greeter-session=example-gtk-gnome
-#greeter-hide-users=false
+greeter-session=lightdm-greeter
+greeter-hide-users=true
 #greeter-allow-guest=true
 #greeter-show-manual-login=false
 #user-session=default
 #allow-guest=true
 #guest-session=UNIMPLEMENTED
-#session-wrapper=lightdm-session
+session-wrapper=/etc/X11/Xsession
 #display-setup-script=
 #greeter-setup-script=
 #session-setup-script=
